acord/models/guild.py

```diff
@@ -121,7 +129,7 @@ class Guild(pydantic.BaseModel, Hashable):
     """ Whether Boosts progress bar is enabled """
     premium_subscription_count: int
     """ Number of guild boosts """
-    premium_tier: int
+    premium_tier: PremiumTierLevel
```

Comment:
:class:\`PremiumTierLevel\`

Reply:
Fixed in https://github.com/Mecha-Karen/acord/pull/11/commits/b8f6ab529a1024a69e5cbc142934dbb22ba31b2c